apps/bundle-size/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "bundle-size",
+  "version": "1.0.0",
+  "private": true,
+  "bin": {
+    "bundle-size": "./bin/bundle-size.js"
+  },
+  "scripts": {
+    "build": "tsc --noEmit",
```

Comment:
It's actually `type-check`, we can change it once CI will run this command

Reply:
it already does -> https://github.com/microsoft/fluentui/blob/master/package.json#L15

Reply:
Nice, updated in eee507e